Develop to Master branch push for CI build configuration and some fixes

AncyRiju opened the pull request.

Commit:
removed sast.yml and added eslint rule

Commit:
Bugg Fix CSS changes

Commit:
ng lint fix

Commit:
ng lint - eqeqeq rule changes

Commit:
ng lint issues fix and none, nil issues

Commit:
doctor update changes

Commit:
issue fixes

Commit:
minor changes

Commit:
Merge branch 'develop' of https://github.com/PSMRI/MMU-UI-NEXT into develop

Commit:
CSS changes

Commit:
bug fixes

Commit:
Merge branch 'develop' of https://github.com/PSMRI/MMU-UI-NEXT into develop

Commit:
CSS changes

Commit:
environtment file changes

Commit:
mmu upgradation chnages

Commit:
Merge branch 'develop' of https://github.com/PSMRI/MMU-UI-NEXT into develop

Commit:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

Commit:
ci build configurations changes

Commit:
lab technician issue

Commit:
case record patch issues

Commit:
ci environmentfile changes

Commit:
Merge branch 'develop' of https://github.com/PSMRI/MMU-UI-NEXT into develop

NKOKKILIGADDA87 committed:
CSS changes

Commit:
doctor update issue fixes

NKOKKILIGADDA87 committed:
CSS changes

Commit:
build changes

drtechie commented on pom.xml:
```diff
@@ -0,0 +1,209 @@
+							<executable>npm</executable>
+							<arguments>
+								<argument>install</argument>
+								<argument>--legacy-peer-deps</argument>
```
This argument was added in the old version because of very old packages.
Probably not needed now.

bhpratibha replied:
@drtechie  Hi Mithun, if we are not adding this argument, the npm installation is failing, Please find the screenshot or the error message while doing the npm install
<img width="960" alt="npm error" src="https://github.com/PSMRI/MMU-UI-NEXT/assets/165631570/39531bb7-509c-4f51-9884-bac1b6df3ea3">

drtechie replied:
I see okay. Okay, then let it be. 
Ignore the same comment in other repos.

drtechie commented on src/environments/environment.ci.ts.template:
```diff
@@ -1,61 +1,62 @@
   parentAPI: `${MMU_API}`,
 
-  INVENTORY_URL: `${INVENTORY_UI}/#/redirin?`,
+  INVENTORY_URL: `${INVENTORY_UI}inventory-ui-next/#/redirin?`,
```
No, we don't want inventory-ui-next here.
Instead {INVENTORY_UI} set in env will have the base URL detail.
Previous implementation (removed line) is correct.

bhpratibha replied:
@drtechie HI Mithun, updated as per comments, Please check

drtechie commented on src/environments/environment.ci.ts.template:
```diff
@@ -1,61 +1,62 @@
   redirInUrl: `/pharmacist/redirin`,
 
-  TELEMEDICINE_URL: `${SCHEDULER_UI}/#/?`,
+  TELEMEDICINE_URL: `${SCHEDULER_UI}schedulerui-v1.0/#/?`,
```
We want to get rid of versions in the URLs. 
It should be driven with env variable {SCHEDULER_UI}.
Later when we change to proper URLs, a code change is not needed.

Similar comment as above.

bhpratibha replied:
@drtechie HI Mithun, updated as per comments, Please check

Commit:
fixed PR comments

Commit:
sonar lint fixes

Commit:
style changes

Commit:
issue fixed

Commit:
sonarlint issue fixed

Commit:
sonarlint issue fixed

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
merge conflect

Commit:
Customization changes

Commit:
customization changes

bhpratibha committed:
resolved merge conflicts

Commit:
bug fixes

Commit:
prescription quantity change

Commit:
added console

Commit:
customization

Commit:
merge conflicts while reverting commit

Commit:
resolved merge conflicts for commit revert - 38eebe3

Commit:
merge conflicts resolution after revert 3694d39

Commit:
resolved merge conflicts after revert commit 75ba1d5

NKOKKILIGADDA87 committed:
revert chnages

NKOKKILIGADDA87 committed:
revert chnages

Commit:
quantity prescribed changes

Commit:
Merge branch 'develop' of https://github.com/PSMRI/MMU-UI-NEXT into develop

Commit:
merge conflicts resolved

NKOKKILIGADDA87 committed:
css minor changes

NKOKKILIGADDA87 committed:
resolved merge conflicts

Commit:
git revert commit issues

Commit:
resolved merge conflicts

NKOKKILIGADDA87 committed:
latest revert changes

bhpratibha committed:
resolved merge conflicts

bhpratibha committed:
resolved the age

Commit:
hide state and district fields

Commit:
rearranged  shared module

Commit:
data sync changes

Commit:
resolved merge conflicts

Commit:
bug fixes

Commit:
bluetooth and vitals value changes

Commit:
merge conflicts resolved

Commit:
small change

Commit:
Merge branch 'customization' of https://github.com/PSMRI/MMU-UI-NEXT into customization

Commit:
exteral changes for customization errors

Commit:
changed shared file path

Commit:
age issue changes

Commit:
minor changes

Commit:
data sync login change

NKOKKILIGADDA87 committed:
CSS CHNAGES

NKOKKILIGADDA87 committed:
CSS CHNAGES

NKOKKILIGADDA87 committed:
CSS CHNAGES

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
Error resolve

NKOKKILIGADDA87 committed:
Error resolve

NKOKKILIGADDA87 committed:
Error resolve

Commit:
login issue fixed

devikasuresh20 committed:
Changes for patching height on patient revisit

ravishanigarapu committed:
Merge pull request #80 from devikasuresh20/develop

AMM-713 Changes for patching height on patient revisit

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
CSS Changes

devikasuresh20 committed:
Changes done for revisit filteration issue

devikasuresh20 committed:
Resolved conflicts

ravishanigarapu committed:
Merge pull request #81 from devikasuresh20/develop

AMM 684 - Revisit Search Issue

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
CSS chnages

devikasuresh20 committed:
Changes done for adding Referral section in General OPD QC

devikasuresh20 committed:
Resolved merge conflicts

Commit:
small changes in http interceptor and workarea for data sync

Commit:
Merge branch 'develop' of https://github.com/PSMRI/MMU-UI into develop

ravishanigarapu committed:
Merge pull request #85 from devikasuresh20/develop

AMM-759 Addition of Referral section in General OPD QC

Commit:
date format issue fixed

Commit:
resolved merge conflicts

Commit:
added moment package for date

devikasuresh20 committed:
Changes done for removing covid vaccination status

devikasuresh20 committed:
Resolved merge conflicts

ravishanigarapu committed:
Merge pull request #88 from devikasuresh20/develop

AMM-773 Removal of Covid vaccination status section

Commit:
css cross icon change

Commit:
Merge branch 'develop' of https://github.com/PSMRI/MMU-UI into develop

NKOKKILIGADDA87 committed:
CSS chnages

Commit:
Backup of Common-UI

Commit:
Merged conflicts resolved and customization code is merged with develop for mmu

NKOKKILIGADDA87 committed:
CSS chnages

NKOKKILIGADDA87 committed:
CSS chnages

NKOKKILIGADDA87 committed:
CSS chnages

NKOKKILIGADDA87 committed:
CSS chnages

Commit:
Change in css for tablet view fir radio button in mmu

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

Commit:
bio changes

NKOKKILIGADDA87 committed:
CSS changes

Commit:
Change in env file for datasync in mmu

drtechie reviewed (changes requested):
Please add proper PR description.
Why does the submodule file changes appear here?

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
Merge branch 'develop' of https://github.com/PSMRI/MMU-UI into develop

Commit:
correct url for daya sync in env file

Commit:
Changes for history Past Obstetric History in  PNC visit category for mmu

Commit:
change in git hub workflows for testing purpose for submodules

Commit:
Added submodule's configuration in git hub workflows

Commit:
Updated README file for mmu

Commit:
Reverted the changes for README for mmu

NKOKKILIGADDA87 committed:
CSS changes

Commit:
Modified README file for submodule common-ui in mmu

KpParth commented:
> Please add proper PR description. Why does the submodule file changes appear here?

Hi @drtechie 
Whenever we are doing changes for common-ui module, as it is merged with mmu/tm/hwc, this PR is containing both modules commit.

Commit:
Changes for http to https for FHIR

Commit:
Change in env.test.ts- http to https for FHIR

anandamideShakyan committed:
[33] - Updated  outputPath in angular.json    

Updated outputPath in angular.json to fix the Forbidden error during wildfly deployment.

drtechie committed:
Merge pull request #101 from anandamideShakyan/develop

[33] - Updated  outputPath in angular.json

Commit:
Resolved eqeqeq warnings from es lint

NKOKKILIGADDA87 committed:
upload file cancel button fix

NKOKKILIGADDA87 committed:
Merge branch 'develop' of https://github.com/PSMRI/MMU-UI into develop

NKOKKILIGADDA87 committed:
ng build CSS files syntex errors fix

NKOKKILIGADDA87 committed:
common_UI deployement

NKOKKILIGADDA87 committed:
readme file changes  and add nvmrc file

NKOKKILIGADDA87 committed:
comman UI wardeployement

NKOKKILIGADDA87 committed:
Change in interceptor service for Swasaa

NKOKKILIGADDA87 committed:
comman UI wardeployement

NKOKKILIGADDA87 committed:
comman UI wardeployement

NKOKKILIGADDA87 committed:
comman UI wardeployement

NKOKKILIGADDA87 committed:
comman UI wardeployement

devikasuresh20 committed:
MMU Service Point related Changes

devikasuresh20 committed:
Merge branch 'PSMRI:develop' into develop

devikasuresh20 committed:
Added minor change in service point screen

devikasuresh20 committed:
Added minor change in service point screen

devikasuresh20 committed:
AMM-943 MMU Service Point Related Changes - addition of Taluk and Village (#105)

* MMU Service Point related Changes

* Added minor change in service point screen

* Added minor change in service point screen

devikasuresh20 committed:
Merge branch 'PSMRI:develop' into develop

devikasuresh20 committed:
Added mandatory condition for Travel History section

devikasuresh20 committed:
Resolved conflicts

devikasuresh20 committed:
AMM-928 Travel History mandatory changes (#109)

* MMU Service Point related Changes

* Added minor change in service point screen

* Added minor change in service point screen

* Added mandatory condition for Travel History section

devikasuresh20 committed:
URL changes using jboss context-root

devikasuresh20 committed:
Merge branch 'PSMRI:develop' into develop

devikasuresh20 committed:
Minor change in angular.json

devikasuresh20 committed:
Minor changes

devikasuresh20 committed:
Merge pull request #110 from devikasuresh20/develop

AMM-976 Remove "ui-next" from URL

devikasuresh20 committed:
Download master and interceptor related changes (#118)

devikasuresh20 committed:
AMM-999 Lab Technician - Investigation screen changes (#125)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

devikasuresh20 committed:
AMM-967 History screens are not loading (#126)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

devikasuresh20 committed:
AMM-1013 ANC menstrual history changes (#129)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

devikasuresh20 committed:
AMM-964 File upload changes (#136)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

gupta-soham committed:
Add ESLint and NPM Audit workflow

drtechie committed:
Merge pull request #137 from gupta-soham/feat/add-lint-audit-workflow

Add ESLint and NPM Audit workflow

devikasuresh20 committed:
AMM-1009 Diagnosis changes (#138)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

devikasuresh20 committed:
AMM-1055 Family History Diabetes Mellitus info message changes (#144)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

devikasuresh20 committed:
AMM-1065 File upload changes (#145)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

devikasuresh20 committed:
AMM-1008 Previous ben visit details changes (#146)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

devikasuresh20 committed:
AMM-1090 NCD Screening Systolic and Diastolic BP Age validation changes (#153)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* NCD screening age validation for systolic and diastolic bp

NKOKKILIGADDA87 committed:
mmu wasa first commit

NKOKKILIGADDA87 committed:
mmu wasa second commit

NKOKKILIGADDA87 committed:
mmu wasa third commit

NKOKKILIGADDA87 committed:
mmu wasa forth commit

NKOKKILIGADDA87 committed:
mmu wasa fifth commit

NKOKKILIGADDA87 committed:
mmu wasa sixth commit

NKOKKILIGADDA87 committed:
mmu wasa seventh commit

NKOKKILIGADDA87 committed:
mmu wasa eighth commit

NKOKKILIGADDA87 committed:
mmu wasa ninth commit

NKOKKILIGADDA87 committed:
jboss url changes

NKOKKILIGADDA87 committed:
gitignore changes

NKOKKILIGADDA87 committed:
remove enckey

NKOKKILIGADDA87 committed:
ngx cookie service version change

devikasuresh20 committed:
AMM-1106 Systolic and Diastolic BP changes (#159)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* NCD screening age validation for systolic and diastolic bp

* Systolic and Diastolic BP changes

sandipkarmakar3 committed:
sonal minor issue fix with encreption key location change

sandipkarmakar3 committed:
conflict fix

devikasuresh20 committed:
AMM-1169 Service point screen changes (#164)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* NCD screening age validation for systolic and diastolic bp

* Systolic and Diastolic BP changes

* service point filteration changes

devikasuresh20 committed:
AMM- 1167 check benID URL changes (#165)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* NCD screening age validation for systolic and diastolic bp

* Systolic and Diastolic BP changes

* service point filteration changes

* checkAvailablBenIDLocalServer URL change

NKOKKILIGADDA87 committed:
remove JWT tokken setItem

NKOKKILIGADDA87 committed:
remove EncKey

devikasuresh20 committed:
AMM-1173 Added CSS files for offline MMU (#166)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* NCD screening age validation for systolic and diastolic bp

* Systolic and Diastolic BP changes

* service point filteration changes

* checkAvailablBenIDLocalServer URL change

* Added CSS files for bootsrap, font-awesone and material icons for offline MMU

* Minor change

sandipkarmakar3 committed:
minor issue resolve

NKOKKILIGADDA87 committed:
sessionStorageEncKey value emity

sandipkarmakar3 committed:
minor fix

sandipkarmakar3 committed:
Merge branch 'mmu-wasa' of https://github.com/PSMRI/MMU-UI into mmu-wasa

Sneha6003 committed:
Update README.md

sandipkarmakar3 committed:
ready to merge with Develop

sandipkarmakar3 committed:
-wasa removed

Asp-irin committed:
assigned port 4202 and updated the local env for local development

Asp-irin committed:
updated admin api url

Asp-irin committed:
updated env values

drtechie committed:
Merge pull request #167 from Asp-irin/develop

Assigned port 4202 and updated all the endpoints in the env for local development

sandipkarmakar3 committed:
Devika coments resolved

devikasuresh20 committed:
Merge pull request #155 from PSMRI/mmu-wasa

Mmu wasa Changes

drtechie committed:
Add LICENSE and CODE_OF_CONDUCT

devikasuresh20 committed:
Previous Ben Visit Details Changes (#168)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* NCD screening age validation for systolic and diastolic bp

* Systolic and Diastolic BP changes

* service point filteration changes

* checkAvailablBenIDLocalServer URL change

* Added CSS files for bootsrap, font-awesone and material icons for offline MMU

* Minor change

* Previous Visit Details Changes

abhijeetw035 committed:
fix: update GitHub Actions with correct Node.js, submodules, and working CI build

abhijeetw035 committed:
added missing properties in environment file and updated actions versions

abhijeetw035 committed:
(fix) : injected FHIR_API_BASE and updated all github action versions and updated .gitignore file

drtechie committed:
Merge pull request #169 from abhijeetw035/bugfix/issue-56

fix: update GitHub Actions with correct Node.js, submodules, and working CI build

abhijeetw035 committed:
(fix) : deleted package-prod.yml and updated package.yml build command

abhijeetw035 committed:
Merge branch 'PSMRI:develop' into bugfix/issue-56

drtechie committed:
Merge pull request #170 from abhijeetw035/bugfix/issue-56

(fix) : deleted package-prod.yml and updated package.yml build command

blazethunderstorm committed:
feat(git-hooks): setup Husky, Commitlint, and Commitizen for standardized commit messages

drtechie committed:
Merge pull request #171 from blazethunderstorm/anirudh

feat(commit-config): add conventional commit standards with commitlint, husky and commitizen

helenKaryamsetty committed:
Feature/test (#172)

* Update environment.ci.ts.template with all extra common-ui urls

* Update environment.ci.ts.template

* removed duplicate environment.ci.ts.template

* Update environment.dev.ts

* Update environment.local.ts

* Update environment.prod.ts

* Update environment.test.ts

Commit:
fix(story): service point district field addition changes

devikasuresh20 committed:
Merge pull request #173 from helenKaryamsetty/develop

Service point district field addition changes

Commit:
fix(story): service point district field addition changes

devikasuresh20 committed:
Amm-1394 Lung assessment mandatory condition changes (#174)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* NCD screening age validation for systolic and diastolic bp

* Systolic and Diastolic BP changes

* service point filteration changes

* checkAvailablBenIDLocalServer URL change

* Added CSS files for bootsrap, font-awesone and material icons for offline MMU

* Minor change

* Previous Visit Details Changes

* Lung assessment mandatory validation changes

helenKaryamsetty committed:
Merge branch 'PSMRI:develop' into develop

devikasuresh20 committed:
Merge pull request #175 from helenKaryamsetty/develop

Auto population for single district

drtechie committed:
feat: ensure url doesn't have version. upgrade version

drtechie committed:
fix: war file artifact names

drtechie committed:
Merge pull request #176 from PSMRI/build-update

feat: ensure url doesn't have version. upgrade version

5Amogh committed:
feat: captcha cecurity addition

5Amogh committed:
feat: captch reste handling post login failure

5Amogh committed:
feat: enabling captcha via env

5Amogh committed:
Merge pull request #177 from PSMRI/develop-amogh

#AMM-1247- feat: VAPT Captcha implementation

5Amogh committed:
fix: type fix for the enable captcha variable (#178)

* fix: reset captcha issue fix

* fix: type fix for the enable captcha variable

devikasuresh20 committed:
AMM-1699 Data sync login changes (#179)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* NCD screening age validation for systolic and diastolic bp

* Systolic and Diastolic BP changes

* service point filteration changes

* checkAvailablBenIDLocalServer URL change

* Added CSS files for bootsrap, font-awesone and material icons for offline MMU

* Minor change

* Previous Visit Details Changes

* Lung assessment mandatory validation changes

* fix: added changes for setting providerServicemapid for data sync

* added common-ui latest commit

vishwab1 committed:
Change forgot password erro in response (#180)

* fix: response change error

* fix: response change error

devikasuresh20 committed:
AMM-1412 Pharmacist worklist paginator changes (#181)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* NCD screening age validation for systolic and diastolic bp

* Systolic and Diastolic BP changes

* service point filteration changes

* checkAvailablBenIDLocalServer URL change

* Added CSS files for bootsrap, font-awesone and material icons for offline MMU

* Minor change

* Previous Visit Details Changes

* Lung assessment mandatory validation changes

* fix: added changes for setting providerServicemapid for data sync

* added common-ui latest commit

* fix: pharmacist worklist paginator changes

* fix: paginator changes in pharmacist

devikasuresh20 committed:
AMM-1574 Search screen edit button changes - added font-awesome (#182)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* NCD screening age validation for systolic and diastolic bp

* Systolic and Diastolic BP changes

* service point filteration changes

* checkAvailablBenIDLocalServer URL change

* Added CSS files for bootsrap, font-awesone and material icons for offline MMU

* Minor change

* Previous Visit Details Changes

* Lung assessment mandatory validation changes

* fix: added changes for setting providerServicemapid for data sync

* added common-ui latest commit

* fix: pharmacist worklist paginator changes

* fix: paginator changes in pharmacist

* fix: added font-awesome package for search screen edit button

snehar-nd committed:
fix: amm-1584 - and amm- 1583 (#183)

* fix: amm-1584

* fix: amm-1583

snehar-nd committed:
amm-1589 , amm-1593 (#184)

* fix: amm-1584

* fix: amm-1583

* fix: amm-1589

* fix: code rabitt fixs

* fix: amm-1593

snehar-nd committed:
Fix for AMM-1585, AMM-1590, AMM-1591, AMM-1592 (#185)

* fix: amm-1584

* fix: amm-1583

* fix: amm-1589

* fix: code rabitt fixs

* fix: amm-1593

* fix: amm-1590 aam-1591

* fix: amm-1592

* fix: removed extra angle bracket

devikasuresh20 committed:
AMM-1575 Added ABHA language variables (#186)

* Download master and interceptor related changes

* Added ngIf condition in mat-form-field instead of mat-select

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Added minor changes in file upload table

* Minor changes in file upload

* Diagnosis changes

* NCD screening family history changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* NCD screening age validation for systolic and diastolic bp

* Systolic and Diastolic BP changes

* service point filteration changes

* checkAvailablBenIDLocalServer URL change

* Added CSS files for bootsrap, font-awesone and material icons for offline MMU

* Minor change

* Previous Visit Details Changes

* Lung assessment mandatory validation changes

* fix: added changes for setting providerServicemapid for data sync

* added common-ui latest commit

* fix: pharmacist worklist paginator changes

* fix: paginator changes in pharmacist

* fix: added font-awesome package for search screen edit button

* fix: added abha variables in language json file

snehar-nd committed:
Inventory redirection (#187)

* fix: amm-1584

* fix: amm-1583

* fix: amm-1589

* fix: code rabitt fixs

* fix: amm-1593

* fix: amm-1590 aam-1591

* fix: amm-1592

* fix: removed extra angle bracket

* fix: inventory redirection issue fixed

vanitha1822 committed:
fix: remove the additional slash in the url to fix the 404 issue

vanitha1822 committed:
Merge pull request #188 from PSMRI/nd/vs/urlchange

Fix the URL correction for doctor update
